Remotion/Web/Development.WebTesting/Accessibility/AccessibilityAnalyzer.cs

```diff
@@ -225,8 +225,10 @@ private bool AxeIsInjected ()
     /// </remarks>
     private void InjectIntoIFrames (string source)
     {
+      var maximumTimeToWaitForFrame = TimeSpan.FromMilliseconds (3000);
       foreach (var frame in WebDriver.FindElements (By.TagName ("iframe")))
       {
+        frame.WaitUntilFrameIsVisible (maximumTimeToWaitForFrame);
```

Comment:
how did we find out that there's two issues, once here and one in DateTimeValue? I don't remember which tests failed so I just assumed it was only in DateTimeValueExntesions that we had to add this guard (right now).

Reply:
The guard in DateTimeValueExtensions stems from some time ago and was Chrome-related. The actual reason this ticket exists is that `AccessibilityBocDateTimeValueTest.DateOnly_ExpandedDatePicker` fails with Firefox >= 85, hence we are waiting for the frame to render. The changes in `ScreenshotBocDateTimeValuePickerExtensions` from this PR are purely to avoid/eliminate code duplication because it's exactly the same workaround, just with a specifiable selector so that the new extension method can be used universally.